### PR TITLE
[ticket/13643] Change the priority of kernel_terminate_subscriber to -10000

### PR DESCRIPTION
**Pull request by Nicofuma**
https://tracker.phpbb.com/browse/PHPBB3-13643

PHPBB3-13643

### CODE REVIEW

File: phpBB/phpbb/event/kernel_terminate_subscriber.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ public function on_kernel_terminate(PostResponseEvent $event)
 	public static function getSubscribedEvents()
 	{
 		return array(
-			KernelEvents::TERMINATE		=> 'on_kernel_terminate',
+			KernelEvents::TERMINATE		=> array('on_kernel_terminate', -10000),
```
Comment (bantu) [inline phpBB/phpbb/event/kernel_terminate_subscriber.php]: There is no case for not running this last, right? So make this something like `-PHP_INT_MAX` or `~PHP_INT_MAX` ?

Comment (bantu) [inline phpBB/phpbb/event/kernel_terminate_subscriber.php]: There doesn't seem to be any tests for passing an array.

Comment (Nicofuma) [inline phpBB/phpbb/event/kernel_terminate_subscriber.php]: updated to ~PHP_INT_MAX

Comment (Nicofuma) [inline phpBB/phpbb/event/kernel_terminate_subscriber.php]: about the tests, you mean for `array('on_kernel_terminate', -10000)` ? It's part of the SYmfony api.

**Pull request merged**